Source/VSProps/Base.props

```diff
@@ -63,6 +63,7 @@
       <ExternalTemplatesDiagnostics>true</ExternalTemplatesDiagnostics>
       <DisableAnalyzeExternal>true</DisableAnalyzeExternal>
       <TreatAngleIncludeAsExternal>true</TreatAngleIncludeAsExternal>
+      <PreprocessorDefinitions>FMT_EXCEPTIONS=0;FMT_ASSERT=__noop;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Does this need to be done for the cmake builds as well?

Reply:
It shouldn't be needed since C++ exceptions are only enabled on Windows build.

Reply:
Oh right, I completely forgot that was a thing.

I added it and tested with Visual Studio + CMake.